README.md

```diff
@@ -1,42 +1,19 @@
 # eslint-config-skyscanner
 
-[![Build Status](https://travis-ci.org/Skyscanner/eslint-config-skyscanner.svg?branch=master)](https://travis-ci.org/Skyscanner/eslint-config-skyscanner) [![Greenkeeper badge](https://badges.greenkeeper.io/Skyscanner/eslint-config-skyscanner.svg)](https://greenkeeper.io/)
+[![Build Status](https://travis-ci.org/Skyscanner/eslint-config-skyscanner.svg?branch=master)](https://travis-ci.org/Skyscanner/eslint-config-skyscanner)
+[![npm version](https://badge.fury.io/js/eslint-config-skyscanner.svg)](https://badge.fury.io/js/eslint-config-skyscanner)
```

Comment:
Added new badge with the latest version published

Reply:
Can you use this instead for version? This would be in line with how we have used it for Backpack and the status page 😄 

```suggestion
[![npm version](https://img.shields.io/npm/v/eslint-config-skyscanner.svg)](https://img.shields.io/npm/v/eslint-config-skyscanner)
```

Reply:
Makes sense, changed.
I used the one in `eslint-config-skyscanner-with-prettier` 😅